src/bezpath.rs

```diff
@@ -336,6 +336,25 @@ impl ParamCurveExtrema for PathSeg {
 }
 
 impl PathSeg {
+    /// Returns a new `PathSeg` describing the same path as `self`, but with
+    /// the points reversed.
+    pub fn reverse(&self) -> PathSeg {
+        match self {
+            PathSeg::Line(Line { p0, p1 }) => PathSeg::Line(Line::new(*p1, *p0)),
+            PathSeg::Cubic(c) => PathSeg::Cubic(CubicBez::new(c.p3, c.p2, c.p1, c.p0)),
```

Comment:
As the most incredibly minor style point, I'd put the match arms in Line, Quad, Cubic order (degree 1, 2, 3).